client/src/components/desk/submitted/SubmittedList.tsx

```diff
@@ -17,6 +17,10 @@ class SubmittedList extends Component<SubmittedListProps> {
     }
 
     render() {
+        if (this.props.hidden) { // This is used to hide the submitted list but still call subscribeToUpdatedRequests
```

Comment:
I remember you added a hidden css class earlier and think this is the way you should be doing hidden components in react rights

Reply:
I ended up doing it this way because I couldn't apply the `hidden` class directly to SubmittedList, because if it's literally not on the page then the code that sets up the subscription won't run... Apollo is weird